packages/db/fauna/resources/Function/findMetrics.js

```diff
@@ -0,0 +1,82 @@
+import fauna from 'faunadb'
+
+const {
+  Function,
+  CreateFunction,
+  Query,
+  Lambda,
+  Count,
+  Exists,
+  If,
+  Update,
+  Sum,
+  Match,
+  Index,
+  Map,
+  Get,
+  Var,
+  Select,
+  Documents,
+  Collection,
+  Paginate
+} = fauna
+
+const name = 'findMetrics'
+const body = Query(
+  Lambda(
+    [],
+    {
+      data: {
+        usersTotal: Count(Documents(Collection('User'))),
+        uploadsTotal: Count(Documents(Collection('Upload'))),
+        contentTotal: Count(Documents(Collection('Content'))),
+        contentTotalBytes: Sum(Match(Index('content_sizes'))),
+        dealsActiveTotal: Count(Match(
+          Index('deals_by_status'),
+          'Active'
+        )),
+        dealsQueuedTotal: Count(Match(
+          Index('deals_by_status'),
+          'Queued'
+        )),
+        pinsTotal: Count(Documents(Collection('Pin'))),
+        pinsTotalBytes: Sum(
+          Select('data', Map(
+            Paginate(
+              Match(
+                Index('pins_content_by_status'),
```

Comment:
Hmm, it feels a bit like `dagSize` should be a property of `Pin` as we really don't know the dag size other than when cluster Pins it.

re #83 

cc @alanshaw

Reply:
It would ease things, but duplicate data, as it should also be in the content

Reply:
We did discuss this but decided against. I think it's appropriate for it to be on the `Content` object. If on the pin we'd need to sync the value across all nodes where it is being pinned...